src/data/bucket/symbol_bucket.js

```diff
@@ -593,7 +598,8 @@ class SymbolBucket implements Bucket {
             // An icon can only have one box now, so this indexing is a bit vestigial...
             const box: CollisionBox = (collisionBoxArray.get(k): any);
             if (box.radius === 0) {
-                collisionArrays.iconBox = { x1: box.x1, y1: box.y1, x2: box.x2, y2: box.y2, anchorPointX: box.anchorPointX, anchorPointY: box.anchorPointY };
+                collisionArrays.iconBox = { x1: box.x1, y1: box.y1, x2: box.x2, y2: box.y2, anchorPointX: box.anchorPointX, anchorPointY: box.anchorPointY, featureIndex: box.featureIndex };
```

Comment:
Is the `featureIndex` stored on `iconBox` used anywhere?

Reply:
Oops! That's vestigial from an earlier implementation.